95.md

```diff
@@ -0,0 +1,75 @@
+NIP-95 - Storage and Shared File
+======
+`draft` `optional` `author:frbitten` 
+
+This NIP allows users to send binary data to be stored in the relay. This data can be small pieces of information or even large files.
+
+The intention is to combine the ease of NOSTR, with many clients being developed and a common communication protocol between all. To enable decentralized file sharing. NOSTR communication must work together with other protocols to get the best out of each technology. For lay users to be able to publish their photo, file, audio on several servers in a simple and easy way.
+
+Nostr event
+------------------
+This NIP specifies the use of the `1064` event type (parameterized replaceable event), having in `content` the binary data that you want to be stored in Base64 format.
```

Comment:
[NIP-01] says that:
> for kind `n` such that 1000 <= n < 10000, events are **regular** [...]

Not parameterized or replaceable.

Which is good. I wouldn't want to quote or reply to some image only to have the author troll by changing it out to something else later.

[NIP-01]: https://github.com/nostr-protocol/nips/blob/master/01.md

Reply:
Yes. We had an argument at the beginning and we changed the kid and I forgot to change the description. But a file cannot be changed. The change means being a new file in a new note.